Microsoft.Recognizers.Text.DateTime/Spanish/Extractors/SpanishMergedExtractorConfiguration.cs

```diff
@@ -10,10 +10,16 @@ public class SpanishMergedExtractorConfiguration : IMergedExtractorConfiguration
         public static readonly Regex AfterRegex = 
             new Regex(@"(despues(\s*de(\s+las?)?)?)", RegexOptions.IgnoreCase | RegexOptions.Singleline);
 
-        //TODO: change this to Spanish if there is same requirement of split from A to B as two time points
+        //TODO: change the following three regexes to Spanish if there is same requirement of split from A to B as two time points
         public static readonly Regex FromToRegex = 
             new Regex(@"\b(from).+(to)\b.+", RegexOptions.IgnoreCase | RegexOptions.Singleline);
 
+        public static readonly Regex SingleAmbiguousMonthRegex =
+            new Regex(@"\b(may|march)\b", RegexOptions.IgnoreCase | RegexOptions.Singleline);
```

Comment:
In following patches, please always put the regexes in the YAML files. We can add notes for which ones may be language-specific.

Reply:
there is no YAML file for Spanish right now. I don't want to reference the EnglishDatetimeDefinition in this class.